File: pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Hight: ");
    }
    while (n < 0 || n > 23);

    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            if (j >= n - 1 - i)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf("  ");
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/mario.py
from cs50 import get_int

# Prompt user for input
while True:
    n = get_int("Height: ")
    # Validate user input
    if(n >= 0 and n <= 23):
        break

# loop itrate throw hight
for i in range(n):

    # Print the first half of the pyramid
    print(" " * (n - i - 1), end="")
    print("#" * (i + 1), end="")

    # Print the gap
    print("  ", end="")

    # Print the second half of the pyramid
    print("#" * (i + 1), end="")
    print()
